test/controllers/project_controller_test.rb

```diff
@@ -410,12 +410,12 @@ def test_add_member_writein
     assert_not(target_user.in_group?(eol_project.user_group.name))
     params = {
       id: eol_project.id,
-      candidate: "dick <Tricky Dick>"
+      candidate: "#{target_user.login} <Should Ignore This>"
     }
     requires_login(:add_members, params, mary.login)
     assert_response(:success)
-    target_user = User.find(target_user.id)
-    assert_not(target_user.in_group?(eol_project.admin_group.name))
+    target_user.reload
+    assert_not(target_user.reload.in_group?(eol_project.admin_group.name))
```

Comment:
Do we need to reload twice?
(I think you could get rid of line 417.)

Reply:
Oops!  Will fix that.